examples/Meeting_for_contacts/Ballerina.toml

```
@@ -0,0 +1,14 @@
+[package]
+org = "thisarar"
+name = "Meeting_for_contacts"
+version = "0.1.0"
+distribution = "2201.10.3"
+
+[build-options]
+observabilityIncluded = true
+
+[[dependency]]
+org = "ballerinax"
+name = "hubspot.crm.engagement.meeting"
+version = "1.0.0"
+repository = "local"
```

```suggestion
repository = "local"

```
Make sure to end your file with new line. Check other places.